src/stancjs/stancjs.ml

```diff
@@ -55,6 +55,10 @@ let stan2cpp model_name model_string is_flag_set flag_val =
           flag_val "max-line-length"
           |> Option.map ~f:int_of_string
           |> Option.value ~default:78 in
+        let deprecation_warnings =
+          if canonicalizer_settings.deprecations then []
+          else Deprecation_analysis.collect_warnings typed_ast in
+        let warnings = warnings @ deprecation_warnings in
```

Comment:
Was the previous absence of `deprecation_warnings` here an oversight?
And why don't you handle removals? Is `stancjs` expected to just allow deprecated features?
Feels very wrong that the only change to stancjs tests is to dismiss a pound-comment warning.

Reply:
Previous absence does seem to be an oversight.

Agreed, this was something I forgot to come back to. Stanc3js currently assumes any errors that occur are specifically of type `Errors.t`, which I didn't want to edit for this purpose, so it will need some tweaking